app/views/comments/_form.html.erb

```diff
@@ -47,17 +47,17 @@
         <div class="editor-image-upload">
           <input type="file" id="image-upload-main" name="file" accept="image/*" style="display:none">
           <button type="button" class="crayons-btn crayons-btn--s crayons-btn--icon-left crayons-btn--ghost-dimmed" onclick="handleImageUpload(event,'main')">
-            <%= inline_svg_tag("image.svg", aria: true, class: "crayons-icon", title: "Image") %>
-            <span class="hidden s:inline-block">Upload image</span>
+            <%= inline_svg_tag("image.svg", aria: true, class: "crayons-icon") %>
+            <span class="hidden s:inline-block" aria-hidden="false">Upload image</span>
```

Comment:
Setting `aria-hidden` false here because visually this is hidden. This label is hidden for smaller screens, so in terms of accessbility this was fine initially, but in the larger screens, with the SVG having a title image, the label came out as "Image Upload Image".